source/_layouts/base.liquid

```diff
@@ -58,20 +58,24 @@
     </p>
     <![endif]-->
 
-    {% typogr %}{% render 'header', alerts: alerts %}{% endtypogr %}
+    {%- unless is_playground -%}
+      {% typogr %}{% render 'header', alerts: alerts %}{% endtypogr %}
+    {%- endunless -%}
 
     <main class="content" id="main-content" itemprop="mainContentOfPage" role="main">
-      {% unless page | isTypedoc %}
+      {% unless (page | isTypedoc) || is_playground %}
```

Comment:
LiquidJS has some annoying limitations, so I've discovered this syntax won't work for two reasons:

1. Parens inside tags are not respected, so values with filters need to be `{% assign %}`ed first and then used.
2. `or` instead of `||` (see https://liquidjs.com/tutorials/operators.html)

So:

```suggestion
      {% assign is_typedoc = page | isTypedoc %}
      {% unless is_typedoc or is_playground %}
```